### ISSUE TITLE
Same namespace "visualization" in tree and list

### ISSUE THREAD

**Issue by Gpanaritis** [CSRT-NTUA/AlgoPlus#33]
When I try to include "algoplus.h" I get the following:
```shell
In file included from include/../classes/tree/avl_tree.h:6,
                 from include/algoplus.h:28,
                 from try.cc:2:
include/../classes/tree/../../visualization/tree_visual/tree_visualization.h:20:13: error: redefinition of ‘void visualization::visualize(std::string&, std::string)’
   20 | inline void visualize(std::string &__generate,
      |             ^~~~~~~~~
In file included from include/../classes/list/circular_linked_list.h:6,
                 from include/algoplus.h:22,
                 from try.cc:2:
include/../classes/list/../../visualization/list_visual/linked_list_visualization.h:20:13: note: ‘void visualization::visualize(std::string&, std::string)’ previously defined here
   20 | inline void visualize(std::string &__generate,
```

**Comment by Gpanaritis** [comment]
Maybe we change the namespace in both of them to be like: "tree_visualization" and everything is ok.

**Comment by spirosmaggioros** [comment]
this is closed as of pr #35
